index.js

```diff
@@ -0,0 +1,32 @@
+var express = require('express');
+var mustache = require('mustache-express');
+var port = 3000;
+var app = express();
+
+var angrybirds = require('./modules/angry_birds');
+
+
+
+app.engine('html', mustache());
+app.set('view engine', 'html')
+
+app.set('views', __dirname + '/views')
+
+app.get('/angrybirds', function(request, response){
+    var mastbird ={
+        birdList: angrybirds
+    }
+    response.render('index', mastbird)
+
+  })
+
+  app.get('/angrybirds/:id', function(request, response){
+    response.render('show');
+  });
+var angrybirdsC = require('./controllers/angry_birds_controller')
```

Comment:
```
app.get('/angrybirds', function(request, response){
    var mastbird ={
        birdList: angrybirds
    }
    response.render('index', mastbird)
   })
   app.get('/angrybirds/:id', function(request, response){
    response.render('show');
  });
```
we use controller to remove this code block from the index.js to angry_birds controller.js so our server code can be simple and clean and meet the separation of concern principle